manticore/platforms/linux.py

```diff
@@ -1790,7 +1819,8 @@ def _stat(self, path, buf, is64bit):
         if is64bit:
             ret = self.sys_fstat64(fd, buf)
         else:
-            ret = self.sys_fstat(fd, buf)
+            # FIXME (theo) is that self.sys_fstat for SLinux?
```

Comment:
Ah! good point, I think it is not a typo, but now that I look more closely, I think sys_fstat actually does belong in Linux and not SLinux'